web/gui/dashboard_info.js

```diff
@@ -4201,7 +4201,35 @@ netdataDashboard.context = {
     },
 
     // ------------------------------------------------------------------------
-    // WMI
+    // WMI (Process)
+
+    'wmi.processes_cpu_time': {
+        info: 'Total CPU utilization. The amount of time spent by the process in <a href="https://en.wikipedia.org/wiki/CPU_modes#Mode_types" target="_blank">user and privileged</a> modes.'
+    },
+    'wmi.processes_handles': {
+        info: 'Total number of <a href="https://learn.microsoft.com/en-us/windows/win32/sysinfo/handles-and-objects" target="_blank">handles</a> the process has open. This number is the sum of the handles currently open by each thread in the process.'
+    },
+    'wmi.processes_io_bytes': {
+        info: 'Bytes issued to I/O operations in different modes (read, write, other). This property counts all I/O activity generated by the process to include file, network, and device I/Os. Read and write mode includes data operations; other mode includes those that do not involve data, such as control operations.'
+    },
+    'wmi.processes_io_operations': {
+        info: 'I/O operations issued in different modes (read, write, other). This property counts all I/O activity generated by the process to include file, network, and device I/Os. Read and write mode includes data operations; other mode includes those that do not involve data, such as control operations.'
+    },
+    'wmi.processes_page_faults': {
+        info: 'Page faults by the threads executing in this process. A page fault occurs when a thread refers to a virtual memory page that is not in its working set in main memory. This can cause the page not to be fetched from disk if it is on the standby list and hence already in main memory, or if it is in use by another process with which the page is shared.'
+    },
+    'wmi.processes_file_bytes': {
+        info: 'Current number of bytes this process has used in the paging file(s). Paging files are used to store pages of memory used by the process that are not contained in other files. Paging files are shared by all processes, and lack of space in paging files can prevent other processes from allocating memory.'
+    },
+    'wmi.processes_pool_bytes': {
+        info: 'Pool Bytes is the last observed number of bytes in the paged or nonpaged pool. The nonpaged pool is an area of system memory (physical memory used by the operating system) for objects that cannot be written to disk, but must remain in physical memory as long as they are allocated. The paged pool is an area of system memory (physical memory used by the operating system) for objects that can be written to disk when they are not being used. Nonpaged pool bytes is calculated differently than paged pool bytes, so it might not equal the total of paged pool bytes.'
```

Comment:
It looks like you wanted to use this [statement](https://systemcenter.wiki/?GetElement=Microsoft.Windows.Client.Vista.OperatingSystem.MemoryPoolNonPagedBytes.Collection&Type=Rule&ManagementPack=Microsoft.Windows.Client.Vista.Monitoring&Version=6.0.7250.0)

> Memory\Pool Nonpaged Bytes is calculated differently than Process\Pool Nonpaged Bytes, so it might not equal Process\Pool Nonpaged Bytes\_Total

but confused nonpaged and paged pools.

Reply:
I got your point Vlad, and it looks like our [source](https://github.com/prometheus-community/windows_exporter/blob/master/docs/collector.process.md) of information could be also wrong. :thinking:

Reply:
I compared both (Link you brought and official documentation from windows exporter) and I observed that they are missing other important information. On the other hand I would like to have something easy for our users to compare, so I am only removing the last sentence that is more problematic.